packages/shared/src/general.ts

```diff
@@ -135,7 +135,7 @@ export const hasChanged = (value: any, oldValue: any): boolean =>
 
 export const invokeArrayFns = (fns: Function[], arg?: any) => {
   for (let i = 0; i < fns.length; i++) {
-    fns[i](arg)
+    arg === undefined ? fns[i]() : fns[i](arg)
```

Comment:
I'm just nitpicking, but I think it would save a bit more bytes here if we use rest parameters instead of a ternary operator here. (`...args: any[]`, `fns[i](...args]`)

Reply:
Thanks for the suggestion, I'll update the code accordingly. Appreciate your attention to detail!